Geometry/ForwardGeometry/src/ZdcHardcodeGeometryLoader.cc

```diff
@@ -7,13 +7,12 @@
 
 typedef CaloCellGeometry::CCGFloat CCGFloat;
 
+/*
```

Comment:
Can these commented out parts be just removed, to keep the code clean?

Reply:
done